test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -13,7 +13,7 @@ public class LocalizationTest
     {
         private static readonly string _applicationPath =
 #if DNX451
-            Path.GetFullPath(Path.Combine("..", "..", "..", "LocalizationWebsite"));
+            Path.GetFullPath(Path.Combine("..", "..", "..", "..", "test", "LocalizationWebsite"));
```

Comment:
I'm a bit confused by your question. Are you asking if this change is needed at all? You linked to the exact line I am changing in this PR